### ISSUE TITLE
Unable to add generic aliexpress bulb

### ISSUE THREAD

**Issue by broyuken** [rospogrigio/localtuya#1553]
## The problem

Trying to add a generic bulb from aliexpress, the local tuya app doesn't find it, and when I put in the ip, id and local key I get the following:
Connection to device succeeded but no datapoints found, please try again. Create a new issue and include debug logs if problem persists.

## Environment

- Localtuya version:  5.2.1
- Home Assistant Core version:  2023.11.1
- [x] Does the device work using the Home Assistant Tuya Cloud component ? 
- [] Does the device work using the Tinytuya (https://github.com/jasonacox/tinytuya) command line tool ? unknown
- [] Was the device working with earlier versions of localtuya ? Which one? 
- [] Are you using the Tuya/SmartLife App in parallel ? 

## Steps to reproduce

1. Try to add the device
2. Manually input ip address, host id and local key
3. Receive message that no DP's found

## DP dump

{
  "result": {
    "properties": [
      {
        "code": "switch_led",
        "custom_name": "",
        "dp_id": 20,
        "time": 1699365428370,
        "value": true
      },
      {
        "code": "work_mode",
        "custom_name": "",
        "dp_id": 21,
        "time": 1699365428370,
        "value": "white"
      },
      {
        "code": "bright_value",
        "custom_name": "",
        "dp_id": 22,
        "time": 1699365428370,
        "value": 1000
      },
      {
        "code": "temp_value",
        "custom_name": "",
        "dp_id": 23,
        "time": 1699365428370,
        "value": 0
      },
      {
        "code": "colour_data",
        "custom_name": "",
        "dp_id": 24,
        "time": 1699365428370,
        "value": "011d03e803e8"
      },
      {
        "code": "scene_data",
        "custom_name": "",
        "dp_id": 25,
        "time": 1699365428370,
        "value": "000e0d0000000000000000c80000"
      },
      {
        "code": "countdown",
        "custom_name": "",
        "dp_id": 26,
        "time": 1699365428370,
        "value": 0
      },
      {
        "code": "music_data",
        "custom_name": "",
        "dp_id": 27,
        "time": 1695572085883,
        "value": ""
      },
      {
        "code": "control_data",
        "custom_name": "",
        "dp_id": 28,
        "time": 1695572085883,
        "value": ""
      },
      {
        "code": "debug_data",
        "custom_name": "",
        "dp_id": 29,
        "time": 1695572085883,
        "value": ""
      },
      {
        "code": "rhythm_mode",
        "custom_name": "",
        "dp_id": 30,
        "time": 1699365428798,
        "value": "AAAAAAA="
      },
      {
        "code": "sleep_mode",
        "custom_name": "",
        "dp_id": 31,
        "time": 1699365429075,
        "value": "AAA="
      },
      {
        "code": "wakeup_mode",
        "custom_name": "",
        "dp_id": 32,
        "time": 1699365429841,
        "value": "AAA="
      },
      {
        "code": "power_memory",
        "custom_name": "",
        "dp_id": 33,
        "time": 1699365430699,
        "value": "AAEBHQPoA+gD6AAA"
      },
      {
        "code": "do_not_disturb",
        "custom_name": "",
        "dp_id": 34,
        "time": 1699365428370,
        "value": false
      },
      {
        "code": "cycle_timing",
        "custom_name": "",
        "dp_id": 209,
        "time": 1699365430494,
        "value": "AAAA"
      },
      {
        "code": "random_timing",
        "custom_name": "",
        "dp_id": 210,
        "time": 1699365429989,
        "value": "AAAA"
      }
    ]
  },
  "success": true,
  "t": 1699367272342,
  "tid": "d5946ff17d7911eea89ae2fd993f75a4"
}

## Provide Home Assistant traceback/logs

```
put your log output between these markers
```

**Comment by broyuken** [comment]
I was able to install tuya local and get it working with version 3.5. It also seems to have configured everything automatically without me needing to configure each and every entity.

**Comment by debdalerichard** [comment]
is there even a 3.5 protocol released yet??
